### ISSUE TITLE
vim support

### ISSUE THREAD

**Issue by martindemello** [bridgetownrb/serbea#8]
Has anyone worked on vim support for serbea? It definitely looks nicer than erb, but vim knowing how to highlight and indent erb files is a lot to give up!

**Comment by jaredcwhite** [comment]
Thanks for the inquiry! I'm not a Vim user myself, so I'm not exactly sure what would entail setting up a plugin for highlighting. I can look into it at some point but I'd love for someone already in the Vim community to take a stab at it.

**Comment by martindemello** [comment]
@jaredcwhite yeah, didn't expect you to provide one yourself :) I just figured if there were other people interested this would be the best place to get traction for it.
